### PR TITLE
LPS-168535 Remove roles button should announce role name when focused

### PR DESCRIPTION
**Pull request by pat270**
https://issues.liferay.com/browse/LPS-168535

@drewbrokke I need some help with [roles.jsp#L59](https://github.com/pat270/liferay-portal/blob/0b09fc9cce9c516e835bb2da53a130b0c7e70661/modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp#L59). It's used in [roles.jsp#L885](https://github.com/pat270/liferay-portal/blob/0b09fc9cce9c516e835bb2da53a130b0c7e70661/modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp#L885) and [roles.jsp#L922](https://github.com/pat270/liferay-portal/blob/0b09fc9cce9c516e835bb2da53a130b0c7e70661/modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp#L922).

I'm trying to get the role's title/name in the title attribute using:
```
title='<%= LanguageUtil.format(request, "remove-x", Liferay.Util.escapeHTML(name)) %>'
```

I'm running into the error:
```
modules/apps/users-admin/users-admin-web/build/jspc/org/apache/jsp/user/roles_jsp.java:428: error: package Liferay does not exist
_jspx_th_clay_button_0.setDynamicAttribute(null, "title",  LanguageUtil.format(request, "remove-x", Liferay.Util.escapeHTML(name)) );
```

How do I translate this?

### CODE REVIEW

**Comment by drewbrokke** [comment]
@pat270 Liferay.Util is not a Java class. Use HtmlUtil.escape instead. There are a lot of usages across different jsp pages you can copy (sorry I'm using my phone or else I'd find an example for you).

**Comment by pat270** [comment]
@drewbrokke Thanks for the info it helped. I ended up tinkering with it and couldn't get it to work due to lack of knowledge. I figured something out, let me know if there's a better way.

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp [hunk 1]
```diff
@@ -854,15 +881,27 @@ currentURLObj.setParameter("historyKey", liferayPortletResponse.getNamespace() +
 					rowColumns.push(Liferay.Util.escapeHTML(groupName));
 				}
 
+				var removeRoleButton = '<%= UnicodeFormatter.toString(removeRoleIcon) %>';
+
+				removeRoleButton = removeRoleButton.replace(
```
Comment (veroglez) [inline modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp:886]: just a suggestion, I think the code could be more readable in this way:
```
removeRoleButton = removeRoleButton.replace(
	'data-rowId=""',
	`data-rowId="\${roleId}"`
).replace(
	'title=""',
	`title="\${Liferay.Util.sub('<liferay-ui:message key="remove-x" />', name)}"`
);
```

**Comment by veroglez** [comment]
LGTM! 🚀

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp [hunk 2]
```diff
@@ -854,15 +881,28 @@ currentURLObj.setParameter("historyKey", liferayPortletResponse.getNamespace() +
...
+					.replace('data-rowId=""', 'data-rowid="' + roleId + '"')
+					.replace(
+						'title=""',
+						'title="' +
+							Liferay.Util.sub(
+								'<liferay-ui:message key="remove-x" />',
+								name
+							) +
+							'"'
+					);
```
Comment (drewbrokke) [inline modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp:896]: Same suggestion here as [here](https://github.com/liferay-usm/liferay-portal/pull/976#discussion_r1113692123).

**Review by drewbrokke — CHANGES_REQUESTED** [review]

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp (hunk 2 above)
Comment (pat270) [reply modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/roles.jsp:882]: @drewbrokke done!

**Comment by pat270** [comment]
ci:test:relevant

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**